app/controllers/admin/discourse_calendar/admin_holidays_controller.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+require "holidays"
+
+module Admin::DiscourseCalendar
+  class AdminHolidaysController < AdminDiscourseCalendarController
+    def index
+      begin
+        holidays = Holidays.year_holidays([params[:id]], Time.current.beginning_of_year)
+      rescue Holidays::InvalidRegion
+        raise Discourse::NotFound
```

Comment:
We should output an `invalid region` error here instead of a 404.

Reply:
Sounds good! :+1: Something like?: 
```
return render_json_error(I18n.t("system_messages.discourse_calendar_holiday_region_invalid"), 422)
```

Also, is `422` what you had in mind? :thinking: :smiley:

Reply:
I just updated this to return a 422 and an error message. :smiley: